### PR TITLE
Updated cockroach docker image to 20.1 and improved main.go

### PR DESCRIPTION
**Pull request by codingconcepts**

### CODE REVIEW

File: example/mage.go [hunk 1]
```diff
@@ -10,8 +10,8 @@ import (
 func init() {
 	targets.ProjectName = "example"
 	targets.ProjectType = "service"
-	targets.DockerComposeDevDependencies = []string{"redis", "cockroach", "minio", "rabbit"}
-	targets.DockerComposeTestDependencies = []string{"cockroach"}
```
Comment (oliverpauffley) [inline example/mage.go:14]: I feel as an example its more useful to clearly show these dependencies can be different?

**Review by oliverpauffley — COMMENTED** [review]
Just a thought on the example.

File: example/mage.go (hunk 1 above)
Comment (zeevallin) [reply example/mage.go:14]: ☝️

**Review by zeevallin — CHANGES_REQUESTED** [review]
I think Ollie is right in his comment. It would be good if we could keep it different and also add some comments to the example explaining it a bit.

File: example/mage.go (hunk 1 above)
Comment (codingconcepts) [reply example/mage.go:14]: Added in 👍

**Pull request merged**